cognite/client/_api/geospatial.py

```diff
@@ -690,11 +694,10 @@ def stream_features(
         payload = {
             "filter": filter or {},
             "output": {"properties": properties, "jsonStreamFormat": "NEW_LINE_DELIMITED"},
+            "allowCrsTransformation": "true" if allow_crs_transformation else "false",
+            "allowDimensionalityMismatch": "true" if allow_dimensionality_mismatch else "false",
```

Comment:
Oh my... why are these strings here... 👀 Not your fault obviously...

What happens if we encode these as normal booleans? Does it work? The API docs indicate so at least.